app/ViewStores.tsx

```diff
@@ -52,7 +53,7 @@ export function ViewStores(props: { onCreateClick: () => void }): ReactElement {
   if (storeList === undefined) {
     return (
       <Wrapper>
-        Loading
+        <LoadingSpinner height={70} width={70}></LoadingSpinner>
```

Comment:
what happens next time we need to have a spinner on a list (eg deals list)? will we have to specify the height and width again?

Reply:
you could accept a `size` prop perhaps, that lets you pick a `small` or `large` spinner that have preset sizes

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/25/commits/e1dfe67a9a9c6508f50bee7e6aaebeb542d6ab7d

Reply:
if you're accepting two special strings you should use a template literal type https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html

otherwise someone could pass "potatoes" into `size` and typescript wouldn't complain

Reply:
also there's a weird import of `serialize` in your commit that I don't think is doing anything

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/25/commits/c4a763cbad75c5f362440b08981162eea4745031

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/25/commits/6e6b483c9fa14a78979f3fc9478c3aa1dc393acf

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/25/commits/d67c11cefb6de39e43472f735b747435db34efbc